app/controllers/api/v2/provisioning_templates_controller.rb

```diff
@@ -130,6 +131,11 @@ def export
         send_data @provisioning_template.to_erb, :type => 'text/plain', :disposition => 'attachment', :filename => @provisioning_template.filename
       end
 
+      api :GET, '/provisioning_templates/global_registration', N_('Render Global Registration template')
```

Comment:
Since this is a special feature, should we use something easy to remember, like `/register`?